docs/source/md/quick-start/why_aim.md

```diff
@@ -0,0 +1,34 @@
+## Why do you need Aim?
+
+
+### Aim vs Tensorboard
```

Comment:
Do we want to do this in docs? Maybe better to leave the comparison for blog-posts?

Reply:
We have had number of requests for this. We need it everywhere lol

Reply:
haha, good point :)